preprocess.py

```diff
@@ -9,71 +9,29 @@
 Data pre-processing: build vocabularies and binarize training data.
 """
 
-import argparse
 from collections import Counter
 from itertools import zip_longest
 import os
 import shutil
 
-
-from fairseq.data import indexed_dataset, dictionary
-from fairseq.tokenizer import Tokenizer, tokenize_line
+from fairseq import options
+from fairseq.data import indexed_dataset
+from fairseq.tasks import TASK_REGISTRY
```

Comment:
Let's add a simple `get_task` method to `fairseq/tasks/__init__.py` to make this import cleaner:
```
def get_task(name):
    return TASK_REGISTRY[name]
```

Reply:
Done https://github.com/pytorch/fairseq/pull/448/commits/b73bb739f94fb3db17a6ba46329765a9642143a8